usecase/taxi/greenAndYellowUrls.txt

```diff
@@ -0,0 +1,125 @@
+https://s3.amazonaws.com/nyc-tlc/trip+data/green_tripdata_2013-08.csv
```

Comment:
can we verify that yellow+green is still 1+ billion? IIRC it is but it's been a while